enhancements/baremetal/baremetal-provisioning-config.md

```diff
@@ -0,0 +1,188 @@
+---
+title: Baremetal IPI Config Enhancement
+authors:
+  - "@sadasu"
+reviewers:
+  - "@deads2k"
+  - "@hardys"
+  - "@enxebre"
+  - "@squeed"
+  - "@abhinavdahiya"
+  - "@dhellmann"
+
+approvers:
+  - "@abhinavdahiya"
+  - "@enxebre"
+  - "@deads2k"
+
+creation-date: 2019-11-14
+last-updated: 2019-11-26
+status: not implemented
+see-also:
+  - 
+replaces:
+  - https://github.com/openshift/enhancements/pull/90
+superseded-by:
+  - 
+---
+
+# Config required for Baremetal IPI deployments
+
+## Release Signoff Checklist
+
+- [*] Enhancement is `implementable`
+- [*] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift/docs]
+
+## Open Questions
+
+1. Which is a preferred name for the new CR "Metal3ProvisioningController" or
+"Metal3Controller"?
+
+2. There is a possibility that the MCO would also have to refer to this new CR
+created in operator scope. Would it be OK for the MCO to access the CR with this
+(limited) scope?
+
+## Summary
+
+The configuration required to bring up a Baremetal IPI (metal3) cluster needs
+to be availabe in a Config Resource. We had submitted an earlier proposal [1]
+to augment the BareMetalInfrastructureStatus CR with these config items. Based
+on feedback we recieved we are revising our proposal to create a new CR for
+these configuration items.
+
+This new CR needs to be accessed by just the Installer and the Machine API
+Operator (MAO) and hence does not need to be created with global scope.
+
+## Motivation
+
+The Baremetal IPI deployments are different from the other platform types currently
+being supported by OpenShift in that there is no underlying cloud controller
```

Comment:
This is a bit confusing to me, wouldn't it be more accurate "in that there is no underlying cloud platform exposing an API as in public clouds e.g aws, until the baremetal operator along with ironic are run exposing an inventory of available hosts as custom resources"?

Reply:
Yes, that is indeed a better way to characterize it.

Reply:
Yes, that is indeed a better way characterize it.

Reply:
Done.